packages/react/src/ActionList/Item.tsx

```diff
@@ -272,20 +292,31 @@ export const Item = React.forwardRef<HTMLLIElement, ActionListItemProps>(
       id: itemId,
     }
 
-    const containerProps = _PrivateItemWrapper ? {role: itemRole ? 'none' : undefined} : menuItemProps
+    const containerProps = _PrivateItemWrapper
+      ? {role: itemRole ? 'none' : undefined, ...props}
+      : // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+        (validRole && {...menuItemProps, ...props, ref: forwardedRef}) || {}
```

Comment:
Curious if there's a better way to do this. ESLint was throwing an odd error that didn't seem to be valid, but I'm not too sure what an alternative would look like. Essentially, if `_PrivateItemWrapper` is true, default to no role, and pass the props onto the wrapper. Whereas if `validRole` is true, pass both props, and ref to the list item (`<li>`). If `validRole` is false, don't provide anything, as this will be handled in `ButtonItemWrapper`.